enforcer-rules/src/site/apt/banDynamicVersions.apt.vm

```diff
@@ -25,8 +25,8 @@
 
 Ban Dynamic Versions
 
-  This rule bans dependencies having versions which require resolving (i.e. dynamic versions which might change with each build and require 
-  lookup of {{{https://maven.apache.org/ref/3-LATEST/maven-repository-metadata/repository-metadata.html}repositoy metadata}}). Dynamic versions are either 
+  This rule bans dependencies having versions that require resolving (i.e. dynamic versions which might change with each build and require
```

Comment:
I'm also fine with reverting the `s/which/that` change. For defining clauses we can use "that". For some extra context, see [here](https://www.dictionary.com/e/that-vs-which/#:~:text=Luckily%20there's%20an%20easy%20way,it's%20restrictive%2C%20so%20use%20that.).